website/content/docs/developers/simulator-mode.md

```diff
@@ -39,8 +40,22 @@ simulator.deactivate("test");
 simulator.shutdown();
 ```
 
-To kill a topology, you could also simply terminate this process.
```

Comment:
"Other interfaces for simulator mode are:" should be " Other lifecycle methods to use with simulator mode are:"